app/components/article-comments.js

```diff
@@ -85,7 +85,7 @@ export default Component.extend(
 		init() {
 			this._super(...arguments);
 
-			this.set('model', ArticleCommentsModel.create({
+			this.set('model', ArticleCommentsModel.create(getOwner(this).ownerInjection(), {
```

Comment:
I believe passing current object instance into a constructor breaks SOLID principle.
Helper method sounds like a valid alternative. Do you think it will simplify the code? We'd need to import that helper..